operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -1757,6 +1757,36 @@ gateway:
         hosts:
           - gateway.gem.example.com
 
+# Rules for the Prometheus Operator
+prometheusRule:
```

Comment:
We have moved serviceMonitor to under metaMonitoring (see https://github.com/grafana/mimir/pull/2236) Please do the same for prometheusRule. This make it more clear that these are not going to be processed by Mimir, rather by some kind of metamonitoring application like Grafana Agent / Prometheus Operator.

Reply:
moved under metaMonitoring